geomesa-hbase/geomesa-bigtable-tools/src/main/scala/org/locationtech/geomesa/bigtable/tools/BigtableRunner.scala

```diff
@@ -44,4 +44,13 @@ object BigtableRunner extends Runner {
     new HBaseStatsTopKCommand,
     new HBaseStatsHistogramCommand
   )
+
+  override def environmentErrorInfo(): Unit = {
+    if (sys.env.get("HADOOP_HOME").isEmpty) {
+      println("Warning: you have not set HADOOP_HOME as an environment variable." +
```

Comment:
instead of printlns here we should stick with the appropriate loggers (applies to all Runners)